tools/lsp/preview/wasm.rs

```diff
@@ -231,6 +241,27 @@ pub fn load_preview(component: PreviewComponent) {
     .unwrap();
 }
 
+pub fn resource_url_mapper(
+) -> Option<Box<dyn Fn(&str) -> Pin<Box<dyn Future<Output = Option<String>>>>>> {
+    let callback = PREVIEW_STATE.with(|preview_state| {
+        preview_state
+            .borrow()
+            .resource_url_mapper
+            .as_ref()
+            .map(|rum| js_sys::Function::from((*rum).clone()))
+    });
```

Comment:
```suggestion
    })?;
```

You can return None if the callback is not set.